Pepita.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciooUno
+{
+    internal class Pepita:Persona
+    {
+        // se crea metodo para pepita tocar flauta
+        public void tocarFlauta()
```

Comment:
tocar flauta must receive a param as String with the song

Reply:
has been corrected